results/migrations/0016_add_results_14_data.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+
+import json
+from itertools import chain
+
+from django.core.files import File
+from django.db import migrations
+from django.utils.text import slugify
+
+from cpm_generic.migration_utils import (add_subpage, get_content_type,
+                                         get_image_model, remove_subpage)
+
+MIGRATION_DIR = os.path.dirname(__file__)
+
+
+def get_jury_data():
+    jury_json = os.path.join(
+        MIGRATION_DIR,
+        '0016_add_results_14_data/jury.json'
+    )
+    return json.load(open(jury_json, 'rb'), 'utf8')
+
+
+def add_jury_member_pages(apps, schema_editor):
+    Image = get_image_model(apps)
+    IndexPage = apps.get_model('cpm_generic.IndexPage')
+    JuryMemberPage = apps.get_model("results.JuryMemberPage")
+    Collection = apps.get_model('wagtailcore.Collection')
+
+    jury_member_page_ct = get_content_type(apps, 'results', 'jurymemberpage')
+
+    collection_id = Collection.objects.filter(depth=1)[0]
+    juryindex_page = IndexPage.objects.get(slug='jury')
+
+    for item in get_jury_data():
+        photo = Image(title=item['title'], collection=collection_id)
+
+        photo_file = os.path.join(MIGRATION_DIR, item['photo'])
+        photo.file.save(
+            name=item['title'] + os.extsep + item['photo_ext'],
```

Comment:
it's better to use `item['photo']` here and get rid of `photo_ext` in `data.json`